tasks/Debian.yml

```diff
@@ -7,9 +7,10 @@
     - "linux-image-extra-virtual-lts-{{ ansible_distribution_release }}"
```

Comment:
Could you remove these old dependencies please! They don't seem to be required anymore so should be cleaned up :)